app/src/services/audioService.ts

```
@@ -157,38 +157,44 @@ export class AudioService {
+            reject(error);
+          },
+        });
+
+        logger.debug(`Speaking: ${text}`);
+      } catch (error) {
+        logger.error('Failed to speak:', error);
+        this.isSpeaking = false;
+        this.processNextSpeechInQueue();
+        reject(error);
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)